frontend/src/employee-frontend/components/ChildInformation.tsx

```diff
@@ -168,15 +165,10 @@ const layouts: Layouts<typeof components> = {
       component: 'pedagogicalDocuments' as keyof typeof components,
       open: false
     },
-    ...(featureFlags.childIncomeEnabled
-      ? [{ component: 'income' as const, open: false }]
-      : []),
     { component: 'fee-alterations', open: false }
   ],
   ['FINANCE_ADMIN']: [
-    ...(featureFlags.childIncomeEnabled
-      ? [{ component: 'income' as const, open: false }]
-      : []),
+    { component: 'income' as const, open: false },
```

Comment:
Should be `open: true` for `FINANCE_ADMIN`, like in person (adult) profile page. 

`as const` is not needed here.